doc-Deployment_Planning_Guide/topics/planning.adoc

```diff
@@ -212,9 +212,48 @@ Providers can be added at any level of tenancy. Once added, a provider is visibl
+.Load balancer configuration
+
+* Configure the load balancer to use sticky sessions. This ensures that when a session is started, all requests for that session are sent to the same worker appliance.
+* Configure the load balancer to test for connectivity using the {product-title_short} ping response page: `https://appliance_name/ping`. The expected reply from the appliance is the text string _pong_. Using this URL is preferable to the appliance login URL as it does not establish a connection to the database. 
+
+.Appliance configuration
+
+By default, {product-title_short} user interface worker appliances store session data in the local appliance’s memcache. When operating behind a load balancer, it is better to store the user session data in the shared database (VMDB) rather than in memcache, which is not shared. 
+
+When using a load balacner, configure appliances that have the *User Interface* role enabled to store session data in the database. As a result, the user does not need to re-login if the load balancer redirects them to an alternative server in case the original user interface worker is unresponsive. 
+
+Configure this using the `session_store` parameter within the advanced settings page in the {product-title_short} user interface:
+
+. From the settings menu, select *Configuration*.
+. Click the *Advanced* tab.
+. Change the `session_store` parameter to `sql` in the following line (the default parameter is `cache`):
++
+----
+:server:
+...
+ :session_store: sql
+----
++
+. Click *Save*. 
+
+Configure the `session_store` parameter to point to `sql` on each user interface appliance behind the load balancer.
+
+ifdef::cfme[]
+See https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.5/html-single/general_configuration/#servers[Advanced Settings] in _General Configuration_ for more information on editing configuration files from the appliance user interface.
+
+Also see https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/deploying_cloudforms_at_scale/#load_balancers[Load Balancers] in the _Deploying CloudForms at Scale_ reference architecture for further information. 
+
+For information on configuring database failover in VMDB appliances, see https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6/html-single/high_availability_guide/[the _High Availability Guide_].
+endif::cfme[]
+
+
+[[load-balancer-ssui-addl-config]]
+==== Additional Configuration for Load Balancers and the Self Service User Interface
+
+Running several appliances behind a load balancer can cause login issues from the Self Service user interface because `memcached`, the service storing the session in memory, is not shared between the appliances. To prevent this, configure your environment using the steps below to run the `memcached` service on one appliance and have it shared with the other appliances.
```

Comment:
Are we talking about appliances in general or is this still aligned with UI appliances behind a load balancer? I would read this as still referencing UI appliances, due to continuity from the above sections.

Reply:
Correct. I've edited the start of this section now to use "Running several user interface worker appliances behind a load balancer" (clunky but clearer anyway)